sql/drop.go

```diff
@@ -95,8 +94,7 @@ func (p *planner) DropDatabase(n *parser.DropDatabase) (planNode, error) {
 //   Notes: postgres allows only the index owner to DROP an index.
 //          mysql requires the INDEX privilege on the table.
 func (p *planner) DropIndex(n *parser.DropIndex) (planNode, error) {
-	var b client.Batch
```

Comment:
what's the reason for those changes? I had the impression you preferred implicit zero values in the past.